### DISCUSSION TITLE
Need help resetting everything

### DISCUSSION THREAD

**Discussion by chasedown26** [Guilouz/Creality-Helper-Script-Wiki#458] [Issues]
Fluid is telling me that KAMP and Moonraker and Octoprint are invalid and when i click restore it cant connect.  I can use my printer through fluid but octoprint isnt working properly.  so I was trying to reset it but I get Permission denied when trying to do /root/installer.sh    so i cant uninstall guppy screen or anything else like it says to.  What do I do?  im on a K1

**Comment by alkattaf** [comment]
You can open the helper tool from (6) you will find new option to reset 
With new update of root , you should full rest and re root again

**Reply by alkattaf** [reply]
https://guilouz.github.io/Creality-K1-Series/firmwares/reset-factory-settings/

**Reply by alkattaf** [reply]
Use this command after ssh 

`echo "all" | nc -U /var/run/wipe.sock`
